### PR TITLE
1246728 domain stats endpoint

### PR DESCRIPTION
**Pull request by Natim**
Based on #363 to be able to use tags with statsd.

### CODE REVIEW

**Comment by Standard8** [comment]
Not looked at this in detail, but you might want to update package.json to remove the old dependency.

**Comment by Natim** [comment]
@Standard8 isn't this the case here? https://github.com/mozilla-services/loop-server/pull/365/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L40

File: test/rooms_test.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var expect = require("chai").expect;
 var addHawk = require("superagent-hawk");
 var supertest = addHawk(require("supertest"));
 var sinon = require("sinon");
+var assert = sinon.assert;
```
Comment (n1k0) [inline test/rooms_test.js]: Not sure this is a good idea as this could be easily mistaken for nodejs `assert` module.

**Comment by n1k0** [comment]
The code looks okay to me, but one more time I don't have much context about our metrics infrastructure to ensure it matches it and properly serves the intended purpose. So r+ conditional to other reviews from people from metrics.

**Pull request merged**